javascript/javascript3/week1/index.js

```diff
@@ -0,0 +1,111 @@
+// Find a cool api
+// Songkick API
+// To get a data from Songkick API we need specify two parameters in link : artist id and api key.
+// Then we can make a request to the API and get a JSON response.
+// Response will be an object which has an object and array "perfomance" with objects inside and "location" and "venue" objects.
+//  We can get all information, which we need, for example : type of event "Concert" or date, location and more.
+//
+// Weather app
+
+const button = document.querySelector(".submit");
+const input = document.querySelector("input");
+const message = document.querySelector(".message");
+const city = document.querySelector(".cityName");
+const temp = document.querySelector(".temperature");
+const img = document.createElement("img");
+const wind = document.querySelector(".windSpeed");
+const description = document.querySelector(".descriptionClouds");
+const sunsetSunrise = document.querySelector(".sunsetSunrise");
+
+const visibility = document.querySelector(".visibility");
+// Function for optional task num 8
+
+let map;
+let latitude;
+let longitude;
+
+function initMap(latitude, longitude) {
+    map = new google.maps.Map(document.getElementById("mapContainer"), {
+        center: { lat: latitude, lng: longitude },
+        zoom: 13,
+    });
+}
+
+function printWeather(data) {
+    // The chosen name
+    city.innerHTML = data.name;
+    // Tepmerature
+    const roundTemperature = Math.round(data.main.temp);
+    temp.innerHTML = ` ${roundTemperature} °`;
+    // Icon for the weather type
+    img.src = `
+     http://openweathermap.org/img/wn/${data.weather[0].icon}@2x.png
+     `;
+    document.querySelector(".mainContainer").appendChild(img);
+    // Wind speed
+    wind.innerHTML = `Wind speed ${data.wind.speed}`;
+    //How clowdy it is
+    description.innerHTML = data.weather[0].description;
+    // When sunrise and sunset is
+    // For sunrise
```

Comment:
Great logic there !! Just a small tip to save some lines : maybe you could create a new function doing this so you don't have to do the same for both sunrise and sunset.🥷

```
// for example and then you only call the function when assigning the innerHTML to your sunrise and sunset
function fixTheTime(input){
  const dateObject = new Date(input * 1000)
  return dateObject.toLocaleString("de-DE", {
    hour: "numeric",
    minute: "numeric",
  });
}

```